tests/db-init/dump.sql

```diff
@@ -20,6 +20,45 @@ SET default_tablespace = '';
 
 SET default_with_oids = false;
 
+--
+-- Name: CSAMReports; Type: TABLE; Schema: public; Owner: hrm
+--
+
+CREATE TABLE public."CSAMReports" (
+    id integer NOT NULL,
+    "createdAt" timestamp with time zone NOT NULL,
+    "updatedAt" timestamp with time zone NOT NULL,
+    "accountSid" character varying(255),
+    "twilioWorkerId" character varying(255),
+    "csamReportId" character varying(255),
+    "contactId" integer
+);
```

Comment:
We may want to add a field for which reporting agency we called (such as IWF or other groups). But I am okay leaving that off for now, since it's an easy migration later.